File: 151002/Redkovskiy/StoryTeller/src/main/java/com/example/storyteller/dto/requestDto/PostRequestTo.java
package com.example.storyteller.dto.requestDto;

import com.example.storyteller.model.Story;
import jakarta.validation.constraints.Size;
import lombok.Data;

@Data
public class PostRequestTo {

    private Long id;

    @Size(min = 2, max = 2048)
    private String content;

    private Long storyId;
}

File: 151004/Golovchuk/Lab1/DC_Lab1/DTO/PostRequestTo.cs
using DC_Lab1.DTO.Interface;

namespace DC_Lab1.DTO
{
    public class PostRequestTo(int Id,int? tweetId, string? Content) : IRequestTo
    {
        public int Id { get; set; } = Id;
        public int? tweetId { get; set; } = tweetId;

        public string? Content { get; set; } = Content;
    }
}
